src/Stryker.CLI/Stryker.CLI/StrykerCLI.cs

```
@@ -174,7 +103,7 @@ private void PrintStrykerASCIIName()
             Console.WriteLine();
         }
 
-        private async Task PrintStrykerVersionInformationAsync()
+        private async void PrintStrykerVersionInformationAsync()
```

Why not Task anymore?

---

Because in this case we will never do anything with the result, and if there is an exception here for any reason we want it to be swallowed.

---

Hmm I see. Maybe not the most explicit way to ignore the exception but probably okay for this case.

---

SonarCloud is also quite angry with us over this 😄